{"kind":"article","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/vigramsdk\/authenticationerror\/localizederror-implementations"]}],"schemaVersion":{"minor":3,"major":0,"patch":0},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/LocalizedError-Implementations"},"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError"]]},"sections":[],"topicSections":[{"anchor":"Instance-Properties","title":"Instance Properties","identifiers":["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/failureReason","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/helpAnchor","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/recoverySuggestion"],"generated":true}],"metadata":{"modules":[{"name":"VigramSDK"}],"role":"collectionGroup","title":"LocalizedError Implementations","roleHeading":"API Collection"},"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"role":"collection","title":"VigramSDK","kind":"symbol","images":[{"identifier":"logo.png","type":"icon"}],"abstract":[],"url":"\/documentation\/vigramsdk","type":"topic","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK"},"logo.png":{"variants":[{"url":"\/images\/com.vigram.VigramSDK\/logo.png","traits":["1x","light"]}],"identifier":"logo.png","type":"image","alt":"Vigram logo."},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/recoverySuggestion":{"kind":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror\/recoverysuggestion","abstract":[],"role":"symbol","type":"topic","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/recoverySuggestion","title":"recoverySuggestion","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/failureReason":{"url":"\/documentation\/vigramsdk\/authenticationerror\/failurereason","type":"topic","kind":"symbol","title":"failureReason","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/failureReason"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError":{"title":"AuthenticationError","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"AuthenticationError"}],"type":"topic","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","url":"\/documentation\/vigramsdk\/authenticationerror","kind":"symbol","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AuthenticationError"}],"abstract":[{"type":"text","text":"AuthenticationError contains custom failure events when connecting to Authentication process."}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/errorDescription":{"kind":"symbol","abstract":[],"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"url":"\/documentation\/vigramsdk\/authenticationerror\/errordescription","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","title":"errorDescription","type":"topic"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/helpAnchor":{"abstract":[],"type":"topic","url":"\/documentation\/vigramsdk\/authenticationerror\/helpanchor","kind":"symbol","role":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/helpAnchor","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"title":"helpAnchor"}}}